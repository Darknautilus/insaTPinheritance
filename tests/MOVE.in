R rectangle1 56 46 108 4536 
L l172v 56 46 108 4536 
C circle1 12 15 45
OA agg1 rectangle1 circle1
Oa agg2 l172v agg1
MOVE rectangle1 -45 70
MOVE rectangle2 40 78
MOVE agg2 -10 24
