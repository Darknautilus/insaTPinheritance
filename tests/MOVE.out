OK
#New object: rectangle1
OK
#New object: l172v
OK
#New object: circle1
OK
#New object: agg1
OK
#New object: agg2
OK
ERR
OK
